core/src/main/java/com/hcl/ecomm/core/config/ProductServiceConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.hcl.ecomm.core.config;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.AttributeType;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name = "Product Service Configuration")
+public @interface ProductServiceConfig {
+
+	String domainNameDefaultValue = "localhost/magento2/rest";
+	String servicePathDefaultValue = "/us/V1/products";
+	String searchFieldDefaultValue = "store_id";
+	String searchFieldValueDefaultValue = "2";
+
+	@AttributeDefinition(name = "DOMAIN NAME", description = "This ia domain name", defaultValue = domainNameDefaultValue, type = AttributeType.STRING)
```

Comment:
@nitheshkumar-hcl : minor spell check - should be is instead of ia

Reply:
Changes are committed.